src/react-components/room/ChatSidebar.js

```diff
@@ -80,6 +80,7 @@ export function ChatInput(props) {
   return (
     <div className={styles.chatInputContainer}>
       <TextAreaInput
+        maxLength={MAX_MESSAGE_LENGTH}
```

Comment:
On the client side I think this check should be sufficient. It will provide the UX to let the user know they have entered too many characters (and we should make sure you cant click send when the form is invalid). I don't think we really need to check in send/receive as well. We ideally should add enforcement on the server side though.